### DISCUSSION TITLE
Inline if conditions

### DISCUSSION THREAD

**Discussion by k0mpreni** [a-h/templ#783] [Q&A] [answered]
Hi,

I have a more styli-ish question:

I have an input checked based on a condition, right now I made it working with 
`if CONDITION {
  checked="true"
}`

Like here: 
`
<input
	type="radio"
	name="my-accordion-2"
	if i == 0 {
		checked="checked"
	}
/>
`

Is there a simpler/more cleaner way to do it? I tried using `templ.KV` but got an error:
`checked={ templ.KV("checked", i == 0) }`
Error: `cannot use templ.KV("checked", i == 0) (value of type templ.KeyValue[string, bool]) as string value in argument to templ.JoinStringErrs`

Thanks!

**Comment by joerdav** [comment, marked as answer]
At the moment I think an `if` statement is your best bet. 

There is another option though..

If you don't care about the value of the attribute you can use [boolean](https://templ.guide/syntax-and-usage/attributes#boolean-attributes) attributes:

```templ
templ check(i int) {
	<input
		type="radio"
		name="my-accordion-2"
		checked?={ i == 0 }
	/>
}
```

Which results in:
```html
<input type="radio" name="my-accordion-2" checked>
```

**Reply by k0mpreni** [reply]
Awesome, this will do the trick thanks!
